classes/controller/AdminController.php

```diff
@@ -297,6 +297,9 @@ class AdminControllerCore extends Controller
 
 	protected $logged_on_addons = false;
 
+	/** @var bool if logged employee has access to AdminImport */
+	protected $canImport = false;
```

Comment:
Sorry, I understood. You wrote "snakeCase" using "camelCase". (snake_case / camelCase)